src/content/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph.mdx

```diff
@@ -145,6 +145,15 @@ Alerts      </td>
 [See all alert-related tutorials](/docs/alerts-applied-intelligence/new-relic-alerts/alerts-nerdgraph/nerdgraph-api-nrql-condition-alerts/)      </td>
     </tr>
 
+ <tr>
+      <td>
+Applied Intelligence     </td>
+
+      <td>
+[Topology for correlating incident events](/docs/apis/nerdgraph/examples/topology-nerdgraph-tutorial)    </td>
```

Comment:
I worry that "Topology for correlating incident events" is tough to understand. Is there a way we can simplify and humanize this language a bit? 

How about something like: "How incidents between your services and infrastructure resources relate to one another"

Or something. I don't love that, but it gets a little closer to what I'd love to see here.

Reply:
i might try to simplify it. The other descriptions for those are quite short.